utils/message-generator/debug

```diff
@@ -0,0 +1,55 @@
+  0,
```

Comment:
Where is this new file used?

Reply:
I needed that file to fix a bug of the MG, I don't know how it is included in the commit. I'll remove it.